pyproject.toml

```diff
@@ -1,17 +1,21 @@
 [project]
-name = "snipster"
+name = "pdc-snipster-cli"
 version = "0.1.0"
-description = "Add your description here"
+description = "Code snippet repository"
 readme = "README.md"
 authors = [
     { name = "Brandon Brown", email = "afrobot@gmail.com" }
 ]
 requires-python = ">=3.13"
 dependencies = [
     "alembic>=1.16.4",
+    "fastapi[all]>=0.116.1",
+    "httpx>=0.28.1",
     "psycopg[binary]>=3.2.9",
     "python-dotenv>=1.1.1",
     "rapidfuzz>=3.13.0",
+    "reflex>=0.8.6",
+    "reflex-hosting-cli>=0.1.55",
```

Comment:
Do we need httpx and reflex for the package? You could consider dependency grouping here.

Reply:
I also see ui/pyproject.toml, so maybe you can remove some here? Not a big deal though, just trying to keep package dependencies lean.